lib/AbiBinProvider.js

```diff
@@ -24,16 +24,30 @@ class OpenSTAbiBinProvider extends AbiBinProvider {
    *
    * @param abiFolderPath Folder path where abi is present.
    * @param binFolderPath Folder path where bin is present.
```

Comment:
Should be marked optional, right? `{string} [abiFolderPath]` and `{string} [binFolderPath]`.

Reply:
Done!